validate/automated/nodectl.md

```diff
@@ -20,141 +20,64 @@ import DocsCards from '@components/global/DocsCards';
   </style>
 </head>
 
-:::danger IMPORTANT
-The **manual instructions** to install a Node on MainNet 2.0 are for a **single Layer design**.  MainNet 2.0 will be requiring that you run both Layer0 and Layer1 as we perform Genesis.  
+## What is nodectl
 
-*This will change in the future*.
-:::
+**nodectl** pronounced node "c" "t" "l", node-kuttle, or node control.
 
-:::success IMPORTANT
-It is highly **recommended** that you use **nodectl** to install your Node!
-:::
+The purpose of this utility is to **make things easier** on you.  
```

Comment:
Can we make this a more descriptive purpose? What does nodectl actually do for me as a node operator? Connect to network, install upgrades, etc., etc.